src/net/sourceforge/kolmafia/ShopRow.java

```diff
@@ -317,4 +328,52 @@ public String toString() {
       return KoLConstants.COMMA_FORMAT.format(this.count) + " Meat";
     }
   }
+
+  // Conversion from ShopRow objects to and from data strings.
+  // KoLmafia data file format is tab-separated fields
+  //
+  // Grandma Sea Monkee's Window	ROW124	crappy Mer-kin mask	aerated diving helmet	pristine fish
+  // scale (3)
```

Comment:
I recognise that this is spotless screwing things up but a shorter row might appear nicer?